src/monster.h

```diff
@@ -4,6 +4,7 @@
 #include "creature.h"
 #include "enums.h"
 #include "int_id.h"
+#include "field.h"
```

Comment:
Doesn't need to be included at all. It is either included via Creature.h (as required for the virtual function). The forward declarations from that header would also work without additional code here.